agreement_dynamic/models/header.py

```diff
@@ -0,0 +1,22 @@
+class Header:
+    def __init__(self):
+        self.value = 0
+
+    def __call__(self):
+        return self.value
```

Comment:
On second thought this is a bit confusing since the others are properties, so let's remove.